src/table.js

```diff
@@ -196,6 +196,22 @@ function sourceCache(loadSource) {
   };
 }
 
+const loadChartDataSource = sourceCache(async (source) => {
+  if (source instanceof FileAttachment) {
+    switch (source.mimeType) {
+      case "text/csv": return source.csv({typed: true});
+      case "text/tab-separated-values": return source.tsv({typed: true});
+      case "application/json": return source.json();
+    }
+   
+    throw new Error(`unsupported file type: ${source.mimeType}`);
+  }
+
+  if (isDataArray(source) && arrayIsPrimitive(source))
+    return Array.from(source, (value) => ({value}));
```

Comment:
I believe we disallow array-of-primitives as a data source type for chart cells, so this shouldn’t be necessary (and the compiler doesn’t support it anyway).

```suggestion
```

Reply:
Thanks for catching!